.changelog/1154.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+creating a hvs app or kv secret should not panic
```

Comment:
_nit_: I'd rather just repeat the PR title here

```suggestion
Fix a panic when creating HVS app or secret
```

Reply:
done